src/explorer/mod.rs

```diff
@@ -204,23 +230,23 @@ impl<'a> GitExplorer {
     }
 
     fn paint_branch(
-        &self,
+        &mut self,
```

Comment:
changed immutable reference to self to mutable reference to self. I also need to change mutable reference to self in `paint_commit_track`